packages/e2e-tests/specs/editor/various/reusable-blocks.test.js

```diff
@@ -22,11 +23,24 @@ const reusableBlockInspectorNameInputSelector =
 	'.block-editor-block-inspector .components-text-control__input';
 
 const saveAll = async () => {
+	const publishButtonSelector =
+		'.editor-post-publish-button__button.has-changes-dot';
+	const ariaDisabled = await page.$eval(
+		publishButtonSelector,
+		( element ) => element.ariaDisabled
+	);
+	expect( ariaDisabled ).not.toBe( 'true' );
+
 	await page.click( '.editor-post-publish-button__button.has-changes-dot' );
 	await page.waitForSelector(
 		'button.editor-entities-saved-states__save-button'
 	);
+
 	await page.click( 'button.editor-entities-saved-states__save-button' );
```

Comment:
I think this could make this method a bit more stable:

```suggestion
	const publishButtonSelector = '.editor-post-publish-button__button.has-changes-dot';
	// Wait for the Publish button to become enabled in case the editor is autosaving ATM:
	const publishButton = await page.waitForSelector( publishButtonSelector + '[aria-disabled="false"]' );
	await publishButton.click();

	const saveButtonSelector = 'button.editor-entities-saved-states__save-button';
	const saveButton = await page.waitForSelector( saveButtonSelector );
	await saveButton.click();
```